load_tests/validation/validate.go

```diff
@@ -229,6 +229,14 @@ func validate_cloudwatch(cwClient *cloudwatchlogs.CloudWatchLogs, logGroup strin
 			}
 		}
 
+		/*
+		 * In testing we have found that CW GetLogEvents results are highly inconsistent
+		 * Re-running validation long after tests shows that fewer events were lost than
+		 * first calculated. So we sleep between calls to ensure we never exceed 1 TPS
+		 * load_test.py also has a sleep before validation runs.
+		 */
+		time.Sleep(1 * time.Second)
```

Comment:
Non block : it will be better have a constant , like` OneSec =  1 * time.Second`